spec/requests/legacy/redirect_spec.rb

```diff
@@ -38,6 +38,63 @@
     end
   end
 
+  describe 'datafile resources' do
+    let(:legacy_dataset_name) { 'a-legacy-name' }
+    let(:datafile) { CSV_DATAFILE.with_indifferent_access }
+
+    let(:dataset) do
+      DatasetBuilder
+        .new
+        .with_legacy_name(legacy_dataset_name)
+        .with_datafiles([datafile])
+        .build
+    end
+
+    context 'when the datafile can not be found' do
+      it '' do
```

Comment:
It would help if you added the 'it' part too, so that the human-readable description is complete.

Reply:
Good spot 😧

Reply:
Fixed with https://github.com/alphagov/datagovuk_find/pull/393/commits/0c3c7b043a2ad443c6a36fda2750d2aa46369f14